docs/RELEASING_KREW.md

```diff
@@ -38,10 +38,15 @@ Krew tags versions starting with `v`. Example: `v0.2.0-rc.1`.
 1. **Commit the changes back:**
 
        git commit -am "Release ${TAG:?TAG required}"
+       
+1. **Push PR and merge changes**: The repository hooks forbid direct pushes to
+   master, so the changes from the previous step need to be pushed and merged
+   as a regular PR.
 
 1. **Tag the release:**
 
     ```sh
+    git reset --hard origin/master    # when the previous merge is done
```

Comment:
this requires a 

    git fetch origin

because after the bot/human merges the commit, the commit number might be different. So it needs to be fetched again from upstream.

Reply:
Yes, better be totally explicit here :+1: